Automattic/sensei
# Add content drip promo banner to settings

Fixes #4759

### Changes proposed in this Pull Request
- Add new section in settings **'content-drip-settings'**
- Refactor  **_render_promo_banner_**
- Fix issue in settings.js (that I've introduced in previous PR)

### Testing instructions

- Open SenseiLM -> Settings

- You should be able to see a new section called "Content drip" : 

<img width="698" alt="image" src="https://user-images.githubusercontent.com/7208249/154197347-3453c50f-78bf-4b98-8edf-dce86296fe10.png">

- When you go to the section you should see the banner: 

<img width="1204" alt="image" src="https://user-images.githubusercontent.com/7208249/154197374-0b2a9d35-170c-4bba-ab73-b252b455b894.png">

- When clicking on the button you should be redirected to: 

[Link](https://senseilms.com/pricing/?utm_source=plugin_sensei&utm_medium=upsell&utm_campaign=settings_content_drip )

- Note that the URL has  **utm_campaign=settings_content_drip**  parameter

- When the content drip plugin is installed you should see view like this: 
<img width="1378" alt="image" src="https://user-images.githubusercontent.com/7208249/154225857-7c2364e1-f73f-4b71-9e42-7078aeebf670.png">

### New/Updated Hooks
New filter hook added: _**sensei_settings_content_drip_hide**_ it can be used to hide the content drip promo banner from settings
